src/hooks/useProtocolMetrics.ts

```diff
@@ -123,27 +132,111 @@ export const useProtocolMetrics = <TSelectData = unknown>(select?: (data: Protoc
-export const useOhmFloatingSupply = () => useProtocolMetrics(metrics => metrics[0].ohmFloatingSupply);
+const QUERY_OPTIONS = { refetchInterval: 60000 }; // Refresh every 60 seconds
+
+export const useMarketCap = (subgraphUrl?: string) =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: subgraphUrl || getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].marketCap, ...QUERY_OPTIONS },
+  );
+
+export const useOhmTotalSupply = (subgraphUrl?: string) =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: subgraphUrl || getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].totalSupply, ...QUERY_OPTIONS },
+  );
+
+export const useGOhmTotalSupply = (subgraphUrl?: string) =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: subgraphUrl || getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].gOhmTotalSupply, ...QUERY_OPTIONS },
+  );
+
+export const useTotalValueDeposited = (subgraphUrl?: string) =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: subgraphUrl || getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].totalValueLocked, ...QUERY_OPTIONS },
+  );
+
+export const useTreasuryMarketValue = (subgraphUrl?: string) =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: subgraphUrl || getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].treasuryMarketValue, ...QUERY_OPTIONS },
+  );
+
+export const useTreasuryLiquidBackingPerOhmFloating = (subgraphUrl?: string) =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: subgraphUrl || getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].treasuryLiquidBackingPerOhmFloating, ...QUERY_OPTIONS },
+  );
+
+// export const useTreasuryLiquidBackingPerGOhm = (subgraphUrl?: string) =>
+//   useMetricsBarLatestOnlyQuery(
+//     { endpoint: subgraphUrl || getSubgraphUrl() },
+//     {},
+//     { select: data => data.protocolMetrics[0].treasuryLiquidBackingPerGOhm, ...QUERY_OPTIONS },
+//   );
+
+export const useOhmCirculatingSupply = (subgraphUrl?: string) =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: subgraphUrl || getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].ohmCirculatingSupply, ...QUERY_OPTIONS },
+  );
+
+export const useOhmFloatingSupply = (subgraphUrl?: string) =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: subgraphUrl || getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].ohmFloatingSupply, ...QUERY_OPTIONS },
+  );
 
 /**
- * @deprecated
- * Please avoid using this method unless you are intentionally trying to read from subgraph.
- * You should typically be using `useOhmPrice` from `src/hooks/usePrices.ts`... as subgraph has an indexing delay
+ * Returns the latest OHM price (in USD).
+ *
+ * This data is fetched from the subgraph, and will not reflect market rates.
+ * Do NOT use this if you need real-time data. Instead, use `useOhmPrice` from
+ * `src/hooks/usePrices.ts`.
+ *
+ * @returns
```

Comment:
another jsdoc nit. I kinda liked having `@deprecated` here because it called your attention to the note.

Reply:
Except this function isn't deprecated, as the dashboard uses it!

Reply:
Except this function isn't deprecated, as the dashboard uses it!